lib/types/raw.js

```diff
@@ -49,7 +50,7 @@ function raw (options) {
     : type
 
   function parse (buf) {
-    return buf
+    return parser(buf)
```

Comment:
We probably don't need two levels of functions here. Just replace parse with a variable.